functional_requirements/requirements.txt

```diff
@@ -1,111 +1,188 @@
+When "brightness" is selected, the user can select circularly between these possible values: 
+	Mid (50% - default) 
+	High (75%)
+	Low (25%)
+	Very High (100%)
 
-	
-Power management (background)
+when "bluetooth" is selected the user can choose to activate o deactivate the bluetooth.
+when "health status check" is selected the user can choose to activate o deactivate the health status check notification.
+when "return to change view menu" is selected, return to "Display Selection" menu and save the current settings.
+
+
+
+PART 3 Storage
+
+REQ 3.1 Storage card
+
+The watch shall provide a memory facility which will retain the information after power-off.
+
+REQ 3.2 Stored data 
+
+The permanent storage shall provide the possibility to store this information:
+
+	- Bluetooth enabled
+	- Brightness setting
+	- Health status check enabled
+
+REQ 3.3 Load data
+
+At power-on the watch shall read the stored data and apply them before enabling the interaction with the user.
+
+
+
+
+PART 4 Background management
+
+REQ 4.1 Battery reading
+
+The watch shall read the battery life status periodically
+
+REQ 4.2 Movement reading
+
+The watch shall monitor the accelerometer periodically to detect movement or lack of thereof.
+The movement is triggered when a force of X Newton is detected from the accelerometer (X is a calibration parameter).
+
+REQ 4.3 Battery failure
+
+If the battery life value is not available, a error is displayed to the user in Clock View.
+
+REQ 4.4 Low-power stand-by mode activation
+
+The watch shall go to low-power stand-by mode when a movement is not detected from accelerometer for at least 5 seconds or the power indicator is <=10% and >2%. 
+
+REQ 4.5 Low-power stand-by mode actions
+
+In Low-power stand-by mode the watch shall perfom these actions in order to reduce power consumption:
+
+	- Trigger motor vibration for 2 sec
+	- Set display brightness to "Low"
+	- Disable Bluetooth
+	- Disable Health status check notification
+	- Turn off display
+
+REQ 4.6 Normal mode activation
+
+The watch shall go to Normal mode when a movement is detected from accelerometer for at least 5 seconds and the power indicator is >10%. 
 
-The clock shall go to low-power stand-by mode when a movement is not detected from accelerometer for at least 5 seconds.
-The clock shall reactivate from low power mode when a movement is detected from acceleromenter.
-The power value is always measured in background and when under 10%, a vibration is sent through actuators and the display change to low brightness. When it returns over 10%, it recovers the saved brightness setting.
+REQ 4.7 Normal mode actions
 
+In Normal the watch shall perfom these actions:
 
+	- Set display brightness to stored setting
+	- Set Bluetooth to stored setting
+	- Set Health status check notification to stored setting
 
+REQ 4.8 Low-power deactivation
 
+The watch shall deactivate when the power indicator is <2%.

```

Comment:
For consistency with state diagram, can we say "battery state of charge" rather than "power indicator" ?

Reply:
fixed